airbyte-webapp/src/components/connection/ConnectionForm/ConnectionConfigurationFormPreview.tsx

```diff
@@ -0,0 +1,64 @@
+import { useFormikContext } from "formik";
+import { FormattedMessage } from "react-intl";
+
+import { FlexContainer } from "components/ui/Flex";
+import { Text } from "components/ui/Text";
+
+import {
+  ConnectionScheduleData,
+  ConnectionScheduleType,
+  NonBreakingChangesPreference,
+} from "core/request/AirbyteClient";
+import { FeatureItem, useFeature } from "hooks/services/Feature";
+
+import styles from "./ConnectionConfigurationFormPreview.module.scss";
+
+export const ConnectionConfigurationFormPreview: React.FC = () => {
+  const { getFieldMeta } = useFormikContext();
+  const allowAutoDetectSchema = useFeature(FeatureItem.AllowAutoDetectSchema);
+
+  const scheduleType = getFieldMeta<ConnectionScheduleType>("scheduleType").value;
+  const scheduleData = getFieldMeta<ConnectionScheduleData>("scheduleData").value;
+  const nonBreakingChangesPreference = getFieldMeta<NonBreakingChangesPreference>("nonBreakingChangesPreference").value;
+
+  const frequency = (
+    <div>
+      <Text size="xs" className={styles.text}>
+        <FormattedMessage id="form.frequency" />:
+      </Text>
+      <Text size="md" className={styles.text}>
+        {scheduleType === ConnectionScheduleType.manual && <FormattedMessage id="frequency.manual" />}
+        {scheduleType === ConnectionScheduleType.cron && (
+          <>
+            <FormattedMessage id="frequency.cron" /> - {scheduleData?.cron?.cronExpression}{" "}
+            {scheduleData?.cron?.cronTimeZone}
+          </>
+        )}
+        {scheduleType === ConnectionScheduleType.basic && (
+          <FormattedMessage
+            id={`form.every.${scheduleData?.basicSchedule?.timeUnit}`}
+            values={{ value: scheduleData?.basicSchedule?.units }}
+          />
+        )}
+      </Text>
+    </div>
+  );
+
+  const nonBreakingChanges = allowAutoDetectSchema && (
+    <div>
+      <Text size="xs" className={styles.text}>
+        <FormattedMessage id="connectionForm.nonBreakingChangesPreference.label" />:
+      </Text>
+      <Text size="md" className={styles.text}>
+        <FormattedMessage id={`connectionForm.nonBreakingChangesPreference.${nonBreakingChangesPreference}`} />
+      </Text>
+    </div>
+  );
```

Comment:
This text feels too faint and small against the background.

Reply:
Yeah, agree, I even used a bigger `font-size: 11px;`, since we don't have a 10px size in `<Text />` component

<img width="1479" alt="image" src="https://user-images.githubusercontent.com/20929344/219659247-36444384-9f8f-4e01-bdc4-187ee8464c6d.png">